### ISSUE TITLE
Runeblade of Baron Riverndare

### ISSUE THREAD

**Issue by punims** [dalaranwow/dalaran-wow#5846]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Runeblade of Baron Riverndare is doing much more healing than it should. 
It would seem like the effect is to increase healing taken by an X percent instead of healing for 19 hp per 5 seconds as it should

**Expected behaviour**:
 https://classic.wowhead.com/spell=17625/unholy-aura - classic value
https://wotlk.evowow.com/?spell=17625 - wotlk value
The effects are clear

**Steps to reproduce the problem**:

1. As a fury warrior Dual Wield two of these swords, enchant one with crusader aura
2. See your healing increased by an insane percent, meaning you can't really die unless there's healing reduction effect.

**Include proofs for this behaviour**

You can research this on several pages like 
https://classic.wowhead.com/spell=17625/unholy-aura
https://wotlk.evowow.com/?spell=17625
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://ibb.co/0rytFvZ
can see it here already being abused

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this will be corrected with the next maintenance.
